.drone.yml

```diff
@@ -0,0 +1,120 @@
+---
+platform:
+  os: linux
+  arch: amd64
+
+# Begin
+clone:
+  git-clone:
+    commands: |
+      set -ex
+      git clone -b ${DRONE_TAG:-$DRONE_BRANCH} $DRONE_REMOTE_URL .
+      if [ x$DRONE_PULL_REQUEST != x ]; then
+          git fetch origin refs/pull/$DRONE_PULL_REQUEST/head
+          EMAIL=ci git merge --no-edit FETCH_HEAD
+      fi
+      git rev-parse HEAD
+    image: "casperlabs/buildenv:latest"
+
+kind: pipeline
+name: build-bors-merge
+
+steps:
+- name: lint-pr
+  commands:
+  - "make lint"
+  image: "casperlabs/buildenv:latest"
+  when:
+    event:
+    - pull_request
+
+- name: test-pr
+  commands:
+  - "rustup target add wasm32-unknown-unknown"
+  - "make test"
+  image: "casperlabs/buildenv:latest"
+  when:
+    event:
+    - pull_request
+    status:
+    - success
+  depends_on:
+  - lint-pr
```

Comment:
these can probably run in parallel?

Reply:
don't think running tests is worth it if lint fails.

Reply:
fair but it will save time